dropletutils-read-10x-counts.R

```diff
@@ -83,7 +83,10 @@ if(!is.na(opt$metadata_files)){
   if(is.na(opt$cell_id_column)){
     stop("Cell id column name for metadata files is not provided.")
   }
-  metadata = lapply(metadata_files, function(x) read.csv(x, sep="\t"))
+  metadata = lapply(metadata_files, function(x) 
+                                    read.csv(x, sep="\t",
+                                    check.names=FALSE,
+                                    stringsAsFactors=FALSE))
```

Comment:
If you're going to go multiline I'd rather you went the whole hog- the halfway house isn't very readable.

```suggestion
  metadata = lapply(metadata_files, function(x) {
    read.csv(
      x,
      sep = "\t",
      check.names = FALSE,
      stringsAsFactors = FALSE
    )
  })
```